frontend/app/.server/domain/multi-channel/case-api-service-default.ts

```diff
@@ -1,20 +1,18 @@
 import type { SinCaseService } from '~/.server/domain/multi-channel/case-api-service';
 import type { SinCaseDto } from '~/.server/domain/multi-channel/sin-case-models';
 import { serverEnvironment } from '~/.server/environment';
-import { AppError } from '~/errors/app-error';
-import { ErrorCodes } from '~/errors/error-codes';
 
 export function getDefaultSinCaseService(): SinCaseService {
   return {
     getSinCases: async (): Promise<SinCaseDto[]> => {
       const response = await fetch(`${serverEnvironment.INTEROP_SIN_REG_API_BASE_URL}/cases`);
-      if (!response.ok) throw new AppError(`Cases not found.`, ErrorCodes.SIN_CASE_NOT_FOUND);
+      if (response.status === 404) return [];
```

Comment:
This still needs to handle when `!response.ok`, it just needs to handle the 404 case first..

ex:

```ts
if (response.status === 404) return [];
if (!response.ok) throw...
```

The reason is that the API can return a 400, a 401, a 403, 5xx.. etc.